.gitignore

```diff
@@ -4,3 +4,4 @@ Gemfile.lock
 *.gem
 
 /pkg
+/coverage
```

Comment:
Can you put a screen shot and some details into the PR description about this dependency and what it looks like when run please.